src/box/box.cc

```diff
@@ -90,7 +90,7 @@
 #include "flightrec.h"
 #include "wal_ext.h"
 
-static char status[64] = "unknown";
+static char status[64] = "not_configured";
```

Comment:
I would like to hear @Mons opinion on this initial value. It is okay for me, but I don't know any similar API in tarantool and would re-check it.

Reply:
like to `not_configured` sounds like before `box.cfg`